third-party/easyloggingpp/src/easylogging++.cc

```diff
@@ -2295,15 +2313,17 @@ void DefaultLogDispatchCallback::dispatch(base::type::string_t&& logLine) {
 void AsyncLogDispatchCallback::handle(const LogDispatchData* data) {
   base::type::string_t logLine = data->logMessage()->logger()->logBuilder()->build(data->logMessage(),
                                  data->dispatchAction() == base::DispatchAction::NormalLog);
-  if (data->dispatchAction() == base::DispatchAction::NormalLog
+  /*if (data->dispatchAction() == base::DispatchAction::NormalLog
```

Comment:
Does it affect current functionality ? 
Let's review it

Reply:
Previous functionality did not use async logger - so these lines are not relevant relative to previous versions held by users.
Regarding changes relative to previous code, if using the async logger, there is a difference, as we wanted to: 
- in the previous code, only logs that must be dispatched to a file were added to the logs' queue. Logs to standard output were simply printed to console. Which means that he async logger was behaving as a sync logger for logs that are to standard output.
The change here is that logs that are targeted to standard output are also added to the logs' queue.